src/Services/Gateway/HpsCreditService.php

```diff
@@ -372,14 +372,15 @@ public function get($transactionId)
     /**
      * @param      $startDate
      * @param      $endDate
+     * @param null $deviceID
      * @param null $filterBy
      *
      * @return array|null
      * @throws \HpsException
      * @throws \HpsGatewayException
      * @throws \HpsInvalidRequestException
      */
-    public function listTransactions($startDate, $endDate, $filterBy = null)
+    public function listTransactions($startDate, $endDate, $deviceId = null, $filterBy = null)
```

Comment:
Let's swap `$deviceId` and `$filterBy` parameters to retain backwards compatibility for potential current users of `$filterBy`.